release/src/Main.hs

```diff
@@ -86,6 +86,10 @@ main = do
       uploadArtifacts <- concatMapM (mavenArtifactCoords optsAllArtifacts) mavenUploadArtifacts
       validateMavenArtifacts releaseDir uploadArtifacts
 
+      let npmPackages =
+            [ "//language-support/ts/daml-ledger:npm_package"
```

Comment:
Maybe always build these to detect typos before an upload?

Reply:
that makes sense!